components/scons/scons.p5m

```diff
@@ -0,0 +1,442 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# cd proto && ( find . -type f | sed 's,^./,file path=,' ; find . -type l -ls | sed -e 's,^.*\./,link path=,' -e 's, -> , target=,' ) >> p5m
+file path=usr/share/man/man1/sconsign.1
+file path=usr/share/man/man1/scons-time.1
+file path=usr/share/man/man1/scons.1
+file path=usr/bin/scons-$(COMPONENT_VERSION).py
+file path=usr/bin/scons-$(COMPONENT_VERSION).pyc
+link path=usr/bin/scons-$(COMPONENT_VERSION) target=scons-$(COMPONENT_VERSION).py
+link path=usr/bin/scons target=scons-$(COMPONENT_VERSION)
+file path=usr/bin/scons-time-$(COMPONENT_VERSION).py
+file path=usr/bin/scons-time-$(COMPONENT_VERSION).pyc
+link path=usr/bin/scons-time-$(COMPONENT_VERSION) target=scons-time-$(COMPONENT_VERSION).py
+link path=usr/bin/scons-time target=scons-time-$(COMPONENT_VERSION)
+file path=usr/bin/sconsign-$(COMPONENT_VERSION).py
+file path=usr/bin/sconsign-$(COMPONENT_VERSION).pyc
+link path=usr/bin/sconsign-$(COMPONENT_VERSION) target=sconsign-$(COMPONENT_VERSION).py
+link path=usr/bin/sconsign target=sconsign-$(COMPONENT_VERSION)
+
+# force a dependency on the Python 2.7 runtime
+depend fmri=__TBD pkg.debug.depend.file=python2.7 \
+       pkg.debug.depend.path=usr/bin type=require
+
+file path=usr/lib/python2.7/site-packages/scons-$(COMPONENT_VERSION)-py2.7.egg-info
```

Comment:
Move python modules in vendor-packages. You can try to pass --install-lib=$(PYTHON_LIB)  to setup.py or transfer files after install or in manifest.